test/e2e/volume_provisioning.go

```diff
@@ -116,8 +117,7 @@ var _ = framework.KubeDescribe("Dynamic provisioning", func() {
 
 	framework.KubeDescribe("DynamicProvisioner", func() {
 		It("should create and delete persistent volumes [Slow]", func() {
-			framework.SkipUnlessProviderIs("openstack", "gce", "aws", "gke")
-
+			framework.SkipUnlessProviderIs("openstack", "gce", "aws", "gke", "glusterfs")
```

Comment:
ah.. I failed to fetch the Provider is mapping to cloudprovider. Thanks!  I will follow above convention for seperating the providers.